pulpcore/cli/rpm/content.py

```diff
@@ -272,6 +270,10 @@ def upload(
     body: Dict[str, Any] = {}
     uploads: Dict[str, IO[bytes]] = {}
     if isinstance(entity_ctx, PulpRpmPackageContext):
+        if kwargs.get("relative_path") is None:
+            pulp_ctx.needs_plugin(PluginRequirement("rpm", min="3.18"))
+        else:
+            ValueError("--relative-path must be provided")
```

Comment:
I intend to move this into `preprocess_entity` once https://github.com/pulp/pulp-cli/pull/544 landed.